### ISSUE TITLE
Target profile page rewrite: Create Mouse Phenotypes detail view

### ISSUE THREAD

**Issue by andrewhercules** [opentargets/platform#528]
### Use Cases

1. As a user of the Platform, I want to see a summary of all of the phenotype categories where a mouse model has identified a particular phenotype belonging to a specific category.

2. As a user of the Platform, I want to see a data table with information about the gene orthologue, the phenotype category, the phenotype label, the allelic composition of the mouse model, and any relevant publications.

3. As a user of the Platform, I want to be able to sort the data table so that I can identify particular orthologues, phenotype categories, or individual phenotypes.

4. As a user of the Platform, I want to be able to download the data table information in JSON, CSV, and/or TSV formats.

### Detail View

![mouse_phenotypes_detail_view](https://user-images.githubusercontent.com/7490258/55226069-0183a280-520c-11e9-9d39-850eaf86d359.png)
**[Full-size version](https://drive.google.com/open?id=1VrggsV8ffCZNo3BWy8hSxPi0qxq4A_5j)**

### Design and Interaction Notes

1. For the detail view description, please use the following text:

``Mouse phenotypes by model associated with GENE-SYMBOL orthologues``

Please replace ``GENE-SYMBOL`` with the HGNC gene symbol value.

2. Above the data table, please create a summary visualisation of all of the phenotype categories, similar to what was created for the Pathways detail view (see #252). If a target has a model where a phenotype has been identified that belongs to a given category, please colour the box of the category in Open Targets Purple - ``#7b196a`` and change the colour of the text to white.

3. For the data table, please use the existing data table design pattern and include the filter/search bars in the ``Mouse gene``, ``Phenotype category``, and ``Phenotype label`` columns. 

### Design Assets

1. [Full-size mockup](https://drive.google.com/open?id=1VrggsV8ffCZNo3BWy8hSxPi0qxq4A_5j)
2. [Open Targets brand palette colour codes](https://drive.google.com/open?id=1d0JDmCznInsANZoQC77gadTf9jO0UDOo)

**Comment by andrewhercules** [comment]
@mirandaio, can we please make two changes?

1. Can we please left-align the content in the ``Allelic composition`` category?

![Screenshot 2019-05-16 at 15 57 08](https://user-images.githubusercontent.com/7490258/57864140-492db000-77f3-11e9-9a35-4850d9c00ac8.png)

2. Within the ``Allelic composition`` column, can we please parse the text and format it to match the text formatting that is shown in the Platform (e.g. [ESR1](https://www.targetvalidation.org/target/ENSG00000091831?view=sec:mouse_phenotypes)) and on [the MGI website](http://www.informatics.jax.org/diseasePortal/popup?isPhenotype=true&markerID=MGI:1352467&header=hematopoietic%20system)?

![Screenshot 2019-05-16 at 15 56 12](https://user-images.githubusercontent.com/7490258/57864223-6bbfc900-77f3-11e9-90aa-d89db9a5d585.png)

**Comment by andrewhercules** [comment]
The conditional block at the topic showing the categories looks good.

If possible, can we format the ``Allelic composition`` data so that it is similar to what we currently have in production? We could left-align the text and then slightly reduce the text for the ``involves: ...`` text - see below for an example:

![Screenshot 2019-06-11 at 10 41 35](https://user-images.githubusercontent.com/7490258/59261677-d7038c00-8c35-11e9-8670-24d0fbfdbc62.png)

**Comment by mirandaio** [comment]
@andrewhercules It should be fixed now.

**Comment by andrewhercules** [comment]
Looks good - thank you @mirandaio! 👍 

Will close this ticket
